cmake/shared/PACE_CodeAnalysis.cmake

```diff
@@ -5,7 +5,7 @@ add_custom_target(analyse
 add_custom_target(analyse-mlint
   COMMENT "- Performing MATLAB analysis (Mlint)..."
   BYPRODUCTS "${CMAKE_CURRENT_BINARY_DIR}/mlint.out"
-  COMMAND ${Matlab_MAIN_PROGRAM} -nodisplay -r "\"addpath('${Herbert_ROOT}/../admin');lint_json({'${CMAKE_SOURCE_DIR}/**/*.m'},'${CMAKE_CURRENT_BINARY_DIR}/mlint.json');exit\""
+  COMMAND ${Matlab_MAIN_PROGRAM} -nodisplay -r "\"addpath('${Herbert_ROOT}/admin');lint_json({'${CMAKE_SOURCE_DIR}/**/*.m'},'${CMAKE_CURRENT_BINARY_DIR}/mlint.json');exit\""
```

Comment:
If you use `-batch` rather than `-r`, there's no need to call `exit`. It also means that we don't need to deal with cases where an error occurs in `mlint_json`. https://www.mathworks.com/help/matlab/ref/matlablinux.html#buknt99-1-option1optionN.

Reply:
When I say "we don't need to deal with cases where an error occurs in `mlint_json`", I mean the Matlab process will be killed. As it is now, it's not clear to me how the Matlab process exits if `lint_json` errors.